docs/gestor_dependencias.md

```diff
@@ -0,0 +1,29 @@
+# Gestor de dependencias
+
+Despues de haber elegido un runtime, en mi caso Node.js, necesitamos un gestor de dependencias. Un gestor de dependencias es una herramienta que nos permite instalar y gestionar las dependencias de nuestro proyecto.
+
+## Criterios de decisión
+
+Para elegir el gestor de dependencias para nuestro proyecto tendremose en cuenta los siguientes criterios:
+
+- ** Soporte: ** El gestor de dependencias debe tener soporte para el runtime que hemos elegido, en nuestro caso Node.js.
+
+- ** Simplicidad: ** El gestor de dependencias debe ser fácil de usar y de entender.
+
+- ** Comunidad: ** El gestor de dependencias debe tener una comunidad que pueda ayudar a resolver problemas.
```

Comment:
La mayoría lo tendrán, salvo que escojáis una herramienta muy nueva; en todo caso, es muy difícil de cuantificar. Sí podéis cuantificar lo que se llama el "bus factor", cuantas personas están trabajando activamente en el mismo.